packages/samples/headless-ssr-commerce/app/layout.tsx

```diff
@@ -7,7 +9,15 @@ export const metadata = {
 export default function RootLayout({children}: {children: React.ReactNode}) {
   return (
     <html lang="en">
-      <body>{children}</body>
+      <body>
+        <h1>Coveo Headless Commerce Next.js</h1>
+        <div style={{display: 'flex', alignItems: 'center', gap: '10px'}}>
```

Comment:
I wouldn't clutter the samples with any stlying unless absolutely necessary We want it to be as bare bone as possible

Reply:
I agree but I think in this case it's necessary. I think the only way to have spaces between <span>s without css is to add a bunch of `&nbsp;&nbsp;&nbsp;&nbsp;`. That would be even more confusing IMO. I don't think this will hurt anyone.